apollo-spaceport/proto/reports.proto

```diff
@@ -0,0 +1,388 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package Report;
```

Comment:
It looks like you had to add this line. Ideally no manual changes should be needed to this file so that it's trivial to keep up to date. There are a number of possible changes here:

- we can add a line like this in the upstream source of this file with some special thing to include it only via a special Rust-specific URL.  (Or we can change the TS version to be happy with `package Report` so we'd only have to hide it for the internal monorepo version. ie we'd just add some sort of BEGIN:EXTERNAL that's the opposite of BEGIN:INTERNAL.)
- you can add a script in this repo that downloads the latest version of reports.proto and inserts this line before saving it
- build.rs can add the line somehow before processing it with tonic_build
- maybe tonic_build has some way to override a package name

Reply:
I'd also request you do update the file to its latest version now, as it doesn't have the latest changes from referenced field reporting.

Reply:
I think the "download + post-process" approach may be cleanest. No point requiring changes to the TS version if we can help it. I'll make sure this issue is addressed sooner.